### PR TITLE
feat: add time series video to the overview page

### PR DESCRIPTION
**Pull request by ChitlangeSahas**
Connects https://github.com/influxdata/ui/issues/4372

Waiting on design to add the videos for the finish page.

Screen Shot 2022-04-19 at 11 15 27 AM

### CODE REVIEW

File: src/homepageExperience/components/steps/Overview.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import React, {PureComponent} from 'react'
+import {SafeBlankLink} from '../../../utils/SafeBlankLink'
```
Comment (ChitlangeSahas) [inline src/homepageExperience/components/steps/Overview.tsx:2]: fixing this...

File: src/homepageExperience/components/steps/Overview.tsx [hunk 2]
```diff
@@ -13,7 +14,24 @@ export class Overview extends PureComponent {
...
+          <iframe
+            width="560"
+            height="315"
+            src="https://www.youtube.com/embed/KZwr1xBDbBQ"
+            title="YouTube video player"
+            frameBorder="0"
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+            allowFullScreen
+            style={{marginBottom: '20px'}}
+          />
```
Comment (ChitlangeSahas) [inline src/homepageExperience/components/steps/Overview.tsx:26]: I got this from youtube's `embed` options.

File: src/homepageExperience/components/steps/Overview.tsx [hunk 3]
```diff
@@ -13,7 +14,24 @@ export class Overview extends PureComponent {
             and write and execute some basic queries.
           </p>
 
-          <p>Without further ado, let’s get started.</p>
+          <iframe
+            width="560"
+            height="315"
+            src="https://www.youtube.com/embed/KZwr1xBDbBQ"
+            title="YouTube video player"
```
Comment (hoorayimhelping) [inline src/homepageExperience/components/steps/Overview.tsx:21]: I think we should change this to something like `InfluxData - What is Time Series`

**Review by hoorayimhelping — APPROVED** [review]
LGTM with one comment about changing the title

File: src/homepageExperience/components/steps/Overview.tsx (hunk 3 above)
Comment (ChitlangeSahas) [reply src/homepageExperience/components/steps/Overview.tsx:21]: Oh yeah good call!

**Pull request merged**